lua/customroles/ghostwhisperer.lua

```diff
@@ -16,21 +16,22 @@ ROLE.shortdesc = "Has a Ghosting Device that they can use on dead players to all
 
 ROLE.team = ROLE_TEAM_INNOCENT
 
-ROLE.convars = {}
-table.insert(ROLE.convars, {
-    cvar = "ttt_ghostwhisperer_ghosting_time",
-    type = ROLE_CONVAR_TYPE_NUM,
-    decimal = 0
-})
-table.insert(ROLE.convars, {
-    cvar = "ttt_ghostwhisperer_limited_chat",
-    type = ROLE_CONVAR_TYPE_BOOL
-})
-table.insert(ROLE.convars, {
-    cvar = "ttt_ghostwhisperer_max_abilities",
-    type = ROLE_CONVAR_TYPE_NUM,
-    decimal = 0
-})
+ROLE.convars = {
+    {
+        cvar = "ttt_ghostwhisperer_ghosting_time",
+        type = ROLE_CONVAR_TYPE_NUM,
+        decimal = 0
+    },
+    {
+        cvar = "ttt_ghostwhisperer_limited_chat",
+        type = ROLE_CONVAR_TYPE_BOOL
+    },
+    {
+        cvar = "ttt_ghostwhisperer_max_abilities",
+        type = ROLE_CONVAR_TYPE_NUM,
+        decimal = 0
+    }
+}
 
 ROLE.translations = {
     ["english"] = {
```

Comment:
GitHub will only let me add comments on lines that were changed, but I think we should also add an `ply.IsRoleAbilityDisabled and ply:IsRoleAbilityDisabled()` check to the PlayerCanSeePlayersChat hook below on line 55

Reply:
Per our discussion on Discord, there isn't a good way to do this since we would ideally want to track the ghostwhisperer that caused this and see if _they_ are disabled, but that's not something we do right now. Also since the target would be dead at this point they can't be disabled after becoming a ghost so we're just going to leave it as-is.